src/ads/ima/requester.js

```diff
@@ -406,40 +406,80 @@ Scoped.define("module:Ads.IMARequester", [
 
             /**
              * @param ad IMA Ad data
-             * @param options
+             * @param dyn
              */
-            _showCompanionAd: function(ad, options) {
+            _showCompanionAd: function(ad, dyn) {
+                var element, playerElement, position, selector, options, height, width;
+                playerElement = dyn.activeElement();
+                options = dyn.get("companion-ad");
+                if (options.split('|').length > 0) {
+                    position = options.split('|')[1] || 'bottom';
```

Comment:
Hi, @sambua! Wouldn't it be better to add a new attribute `companion-ad-position` on the player with `bottom` as default value? It just seems more self explanatory than having to add `|` and a value in the `companion-ad` attribute.